gdal/nmake.opt

```diff
@@ -767,6 +767,13 @@ OCI_INCLUDE =	-I$(ORACLE_HOME)\oci\include
 #OPENJPEG_CFLAGS = -IC:\openjpeg\include\openjpeg-2.X
 #OPENJPEG_LIB = C:\openjpeg\lib\openjp2.lib
 
+# Uncomment for Grok support
+#GROK_ENABLED = YES
+# Note: this must point to the
+# directory where grok.h and grk_config.h are set
+#GROK_CFLAGS = -IC:\grok\include\grok-X
+#OPENJPEG_LIB = C:\grok\lib\grokj2k.lib

```

Comment:
Add a GROK_PLUGIN = YES option to build as plugin. See frmts/makefile.vc and frmts/pdf/makefile.vc for additional places where to add that support / take inspiration from

Reply:
Thanks, as commented above, we can wait on this.